src/stores/video.ts

```diff
@@ -230,30 +231,61 @@ export const useVideoStore = defineStore('video', () => {
   })
 
   // Routine to make sure the user has chosen the allowed ICE candidate IPs, so the stream works as expected
-  const iceIpCheckInterval = setInterval(() => {
-    // Pass if there are no available IPs yet or if the user has already set the allowed ones
-    if (availableIceIps.value === undefined || !allowedIceIps.value.isEmpty()) {
+  let warningTimeout: NodeJS.Timeout | undefined = undefined
+  const iceIpCheckInterval = setInterval(async (): Promise<void> => {
+    // Pass if there are no available IPs yet
+    if (availableIceIps.value === undefined) return
+
+    // Cancel the check if the user has already set the allowed ICE IPs
+    if (!allowedIceIps.value.isEmpty()) {
+      clearInterval(iceIpCheckInterval)
+      clearTimeout(warningTimeout)
       return
     }
-    // If there's more than one IP candidate available, send a warning an clear the check routine
+
+    // If there's more than one IP candidate available, try getting information about them from BlueOS. If not
+    // available, send a warning an clear the check routine.
     if (availableIceIps.value.length >= 1) {
-      Swal.fire({
-        html: `
-          <p>Cockpit detected more than one IP address being used to route the video streaming. This often
-          leads to video stuttering, especially if one of the IPs is from a non-wired connection.</p>
-          </br>
-          <p>To prevent issues and achieve an optimal streaming experience, please:</p>
-          <ol>
-            <li>1. Open the video configuration page (Main-menu > Configuration > Video).</li>
-            <li>2. Select the IP address that should be used for the video streaming.</li>
-          </ol>
-        `,
-        icon: 'warning',
-        customClass: {
-          htmlContainer: 'text-left',
-        },
-      })
-      clearInterval(iceIpCheckInterval)
+      try {
+        const ipsInfo = await getIpsInformationFromVehicle(globalAddress)
+        ipsInfo.forEach((ipInfo) => {
+          const isIceIp = availableIceIps.value.includes(ipInfo.ipv4Address)
+          const alreadyAllowedIp = allowedIceIps.value.includes(ipInfo.ipv4Address)
+          if (ipInfo.interfaceType !== 'WIRED' || alreadyAllowedIp || !isIceIp) return
+          console.info(`Adding the wired address '${ipInfo.ipv4Address}' to the list of allowed ICE IPs.`)
+          allowedIceIps.value.push(ipInfo.ipv4Address)
```

Comment:
I'm late for the party, but just for future note, this IPv4 thing is a nightmare. I know BlueOS doesn't use IPv6, but nothing is helping to debug when the ice strategy fails because it's using IPv6.

Reply:
Hey Joao, could you detail the situation where this is creating a problem? Are IPv6 addresses being offered on the allowed list right now or are you talking about manual inputed ones?

Reply:
There are no restrictions on the MCM side, so when the network uses IPV6, ICE will use it, and Cockpit will not be able to use it, right?

Reply:
Got it!
I'm actually not sure.
Is beacon providing information on IPv6 addresses?
Can you check at [http://blueos.local/beacon/v1.0/services](http://blueos.local/beacon/v1.0/services)? I don't have IPv6 here to check.